src/Storefront/Framework/SystemCheck/SalesChannelsReadinessCheck.php

```diff
@@ -86,20 +85,4 @@ private function doRun(): Result
             $extra
         );
     }
-
-    /**
-     * @return array<string>
-     */
-    private function fetchSalesChannelDomains(): array
```

Comment:
good, that we talked about this :grin: 

https://github.com/shopware/shopware/pull/9530#discussion_r2112036825

Reply:
:D